src/kakarot/instructions.cairo

```diff
@@ -622,6 +623,11 @@ namespace EVMInstructions {
                 }
                 return ctx;
             } else {
+                let is_precompile = Precompiles.is_precompile(address=ctx.evm_contract_address);
```

Comment:
here, calling a precompile address would make 
```
let (bytecode_len) = IEvmContract.bytecode_len(
                    contract_address=ctx.starknet_contract_address
                );
```
error out?

Reply:
yes, when the `contract_address` does not exist, cairo raises